draft-ramseyer-grow-peering-api.md

```diff
@@ -350,11 +290,96 @@ The client will send a structure along with the request, as follows:
 
 The server then responds with the same structure, with the information that it understands (status, etc).
 
-COMPLETION      {#completion}
-----------
+### COMPLETION      {#completion}
 If both sides report that the session is established, then peering is complete.
 If one side does not configure sessions within the server's acceptable configuration window (TimeWindow), then the server is entitled to remove the configured sessions and report "Unestablished" to the client.
 
+Authentication
+==============
+
+Authentication to this API is performed with Bearer tokens. Two methods are defined below to obtain a bearer token,
+the choice of which is supported is up to operator preference however the authors recommend the use of RPKI Signed
+Checklists over reliance on an external OIDC provider.
+
+OAuth with RPKI Signed Checklists
+---------------------------------
+
+This section of the document defined an extension to the OAuth 2.0 protocol ({{!RFC6749}}) to allow for a client to
+obtain an access token using RPKI Signed Checklists ({{!RFC9323}}).
```

Comment:
I'd replace the reference to RFC9323 with a **normative** reference to `draft-blahaj-sidrops-rsm`

Reply:
Done.